sunpy/timeseries/sources/eve.py

```diff
@@ -57,35 +57,63 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, axes=None, title="EVE/ESP Level 1", **kwargs):
         """
+        Plots the EVE ESP Level 1 timeseries data.
+
         Parameters
         ----------
+        axes : `matplotlib.axes.Axes`, optional
+            The axes on which to plot the TimeSeries. Defaults to current axes.
         title : `str`, optional
             Plot title.
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
+
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
         """
 
         self._validate_data_for_plotting()
 
-        names = ('Flux \n 0.1-7nm', 'Flux \n 18nm', 'Flux \n 26nm', 'Flux \n 30nm', 'Flux \n 36nm')
+        if axes is None:
```

Comment:
So this function will do nothing if you provide an axes?

Reply:
We might need some plot tests after all.

Reply:
We'd need to provide it with a list of axes, should I mention that in the docstring?

Reply:
Maybe.

Reply:
As a note? Or should under parameters would do?

Reply:
Would a list of axes change anything?

Reply:
This source in particular requires a list of axes right so not passing a list would return an index out of range error, I suppose I'd have to handle that error in the code?

Reply:
Sure. I think we need plot tests for each source to ensure it works as we want. They won't need to be figure tests. We can leave that to peek.

Reply:
How would we test these plots without a figure test?

Reply:
We check the inputs work as we expect them to?

Reply:
I'm a bit confused as to how exactly we'd be testing the axes input. 
How would we vary the inputs to test?

Reply:
Its fine, lets not test this.